rpcs3/Emu/RSX/Program/GLSLCommon.cpp

```diff
@@ -173,15 +142,15 @@ namespace glsl
 			switch (f)
 			{
 			case COMPARE::SEQ:
-				return Op0 + " == " + Op1;
+				return fmt::format("CMP_FIXUP(%s) == CMP_FIXUP(%s)", Op0, Op1);
 			case COMPARE::SGE:
-				return Op0 + " >= " + Op1;
+				return fmt::format("CMP_FIXUP(%s) >= CMP_FIXUP(%s)", Op0, Op1);
 			case COMPARE::SGT:
-				return Op0 + " > " + Op1;
+				return fmt::format("CMP_FIXUP(%s) > CMP_FIXUP(%s)", Op0, Op1);
 			case COMPARE::SLE:
-				return Op0 + " <= " + Op1;
+				return fmt::format("CMP_FIXUP(%s) <= CMP_FIXUP(%s)", Op0, Op1);
 			case COMPARE::SLT:
-				return Op0 + " < " + Op1;
+				fmt::format("CMP_FIXUP(%s) < CMP_FIXUP(%s)", Op0, Op1);
```

Comment:
why not format SNE the same way?

Reply:
copypasta

Reply:
fixed

Reply:
lol. I didn't even see the missing return